Alignment/CocoaModel/src/CocoaDaqReaderRoot.cc

```diff
@@ -128,7 +129,7 @@ OpticalAlignMeasurementInfo CocoaDaqReaderRoot::GetMeasFromPosition2D( AliDaqPos
   OpticalAlignMeasurementInfo meas;
   
   meas.type_ = "SENSOR2D";
-  meas.name_ = pos2D->GetID();
```

Comment:
@davidlt wouldn't it be sufficient to write this:
```
meas.name_ = pos2D->GetID().Data();
```
as the compiler knows how to convert a char to string?
Naively I would expect if we assign to a `std::string` the rhs will be implicitely converted to this type w/o going through another type. Or does the compiler optimize code such that it changes the type of `meas.name_` from `std::string` to `std::string_view`?